geoportailv3/static/js/layermanager/layermanagerdirective.js

```diff
@@ -45,4 +45,23 @@ app.LayermanagerController.prototype.removeLayer = function(layer) {
 };
 
 
+/**
+ * @param {ol.layer.Layer} layer Layer.
+ * @export
+ */
+app.LayermanagerController.prototype.changeVisibility = function(layer) {
+  var currentOpacity = parseFloat(layer.get('opacity'));
```

Comment:
Also, I don't think we should use a `parseFloat` here. This sounds like a bug in ngeo.